CITATION

```diff
@@ -0,0 +1,29 @@
+If you use iminuit for work/research presented in a publication, we recommend and encourage to add the following two references:
+
+Piti Ongmongkolkul and Christoph Deil and Hans Dembinski and David Men\'endez Hurtado and Chris Burr and Andrew ZP Smith and Fabian Rost and Alex Pearce and Lukas Geiger and Omar Zapata, iminuit - A Python interface to MINUIT, 2012 --, https://github.com/iminuit/iminuit; accessed online <your current date>.
```

Comment:
Suggest to avoid the names here.
It'll already be painful to maintain the list of names in the bibtex entry below, but duplicating it here on this line just make it harder.

For presentations, I think we'll rather encourage people to put the logo / URL.

For papers, I think iminuit will usually appear in the acknowledgements, along with other sofware and services used.
How about suggesting a sentence to put, like they do e.g. in http://www.astropy.org/acknowledging.html ?

For us it could be:

> This research made use of MINUIT \cite{1975CoPhC..10..343J} via the iminuit \cite{iminuit} Python interface.

Or if you want to put iminuit first, it could be:

> This research made use of iminuit \cite{iminuit}, a Python interface to MINUIT \cite{1975CoPhC..10..343J}.

Reply:
Good point, I agree.